src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Scanner;
+
+
+public class Storage {
+    private final String filePath;
+
+    Storage(String _filePath) {
+        this.filePath = _filePath;
+    }
+
+    public ArrayList<Task> load() throws FileNotFoundException, ParseException {
+        ArrayList<Task> data = new ArrayList<>();
+
+        File f = new File(this.filePath);
+        Scanner s = new Scanner(f);
+
+        while (s.hasNext()) {
+            String[] split = Arrays.stream(s.nextLine().split("\\|")).map(String::trim).toArray(String[]::new);
```

Comment:
can consider like splitParts or similar names to make it more descriptive, split is not really a noun.